app/controllers/your-psp/your-psp-tasks.service.test.js

```diff
@@ -0,0 +1,153 @@
+const { expect } = require('chai')
+const { getCurrentCredential } = require('../../utils/credentials')
+const { getStripeTaskList, stripeTaskListIsComplete } = require('./your-psp-tasks.service')
+const gatewayAccountFixtures = require('../../../test/fixtures/gateway-account.fixtures')
+
+describe('Progress of Stripe PSP tasklist page', () => {
+  it('should correctly calculate all conditions being met for Stripe',  () => {
```

Comment:
Can we be more specific about what we are testing? 

```suggestion
  it('should calculate Stripe tasks statuses correctly when all tasks are completed', () => {
```